lms/djangoapps/grades/signals/handlers.py

```diff
@@ -123,6 +122,7 @@ def enqueue_grade_update(sender, **kwargs):  # pylint: disable=unused-argument
     """
     Handles the SCORE_CHANGED signal by enqueueing an update operation to occur asynchronously.
     """
+    from ..tasks import recalculate_subsection_grade
```

Comment:
Does this import still need to be at the function level?  It looks like tasks is only importing from signals.signals now.